tests/components/combined_system_message/combined_system_message.test.jsx

```diff
@@ -2,24 +2,30 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import {shallow} from 'enzyme';
 
-import {Posts} from 'mattermost-redux/constants';
+import {General, Posts} from 'mattermost-redux/constants';
+
+import {shallowWithIntl} from 'tests/helpers/intl-test-helper';
 
 import CombinedSystemMessage from 'components/post_view/combined_system_message/combined_system_message.jsx';
 
 describe('components/post_view/CombinedSystemMessage', () => {
+    function emptyFunc() {} // eslint-disable-line no-empty-function
     const baseProps = {
+        teammateNameDisplay: General.TEAMMATE_NAME_DISPLAY.SHOW_USERNAME,
         currentUserId: 'current_user_id',
+        currentUsername: 'current_username',
         allUserIds: ['added_user_id_1', 'user_id_1'],
+        allUsernames: [],
         messageData: [{actorId: 'user_id_1', postType: Posts.POST_TYPES.ADD_TO_CHANNEL, userIds: ['added_user_id_1']}],
         actions: {
-            getMissingProfilesByIds: () => {},  // eslint-disable-line
+            getProfilesByIds: emptyFunc,
```

Comment:
You can also use `jest.fn()` here

Reply:
Yes but recently I'm avoiding to use that whenever there's no actual need to spy or mock a method.